src/test/test-quota-util.c

```diff
@@ -0,0 +1,71 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <errno.h>
+#include <sys/mman.h>
+#include <sys/quota.h>
+
+#include "blockdev-util.h"
+#include "device-util.h"
+#include "quota-util.h"
+#include "tests.h"
+
+#define NUMINTS  (1000)
+#define FILESIZE (NUMINTS * sizeof(int))
+
+int *file_nmapped_addr(const char *path);
+
+TEST(quotactl_devnum) {
+        _cleanup_free_ char *devnode = NULL;
+        int r;
+        int fd;
+        dev_t devnum;
+        int *m_addr;
+        const char *path = "/tmp/mmapped.bin";
+
+        int id = getuid();
+        ASSERT_OK(id);
+
+        fd = open (path, O_RDWR | O_CREAT | O_TRUNC, 0777 );
```

Comment:
Please drop unnecessary space before `(` and `)`.
Also, please do not use 0777 but use some more restricted mode.